DeHashed API Migration (#44218)
demisto/content: Packs/DeHashed/Integrations/DeHashed/DeHashed_description.md

## DeHashed.com

> This integration allows you to check if your personal information such as your email, username, or password is being compromised.

Reviewer: ```suggestion
This integration checks if personal information, such as emails, usernames, or passwords, has been compromised.
```

### Authentication

> - In the "API Key" field, enter your API key.

Reviewer: ```suggestion
- In the **API Key** field, enter your API key.
```

- To obtain an API key, register at [DeHashed.com](https://www.dehashed.com) and visit the [API documentation page](https://app.dehashed.com/documentation/api).

### Note

> In order to test the instance configuration, the "Test" button passes your API credits.

Reviewer: ```suggestion
Click the **Test** button to validate the instance configuration by checking the available API credits.
```
